FormBuilder.php

```diff
@@ -51,6 +51,10 @@ class FormBuilder extends Widget
     private $view;
 
     private $arrayToTypeFunction;
+	
```

Comment:
@marcosmessias Thanks for your PR! Could you reformat the code to use 4 spaces as indentation, please?